website/members/logic.py

```diff
@@ -143,6 +143,14 @@ def create_recurring_payments(recurring_records, custom_fee=None):
         else:
             count_without_new_payments += 1
         for payment_info in remaining_payments:
+            # use current category fee as custom fee (unless the user forced other one) to
+            # ensure always *1* quota is recorded; however verify that current amount is
+            # the same as the member's category fee and show a warning if needed
+            if custom_fee is None:
+                custom_fee = payment_info['amount']
+                if custom_fee != member.category.fee:
+                    logger.warning(
+                        "Payment with strange amount for member %s: %s", member, payment_info)
```

Comment:
If the custom_fee is enforced by the user this warning won't trigger. Is that ok?

Reply:
Yes! The user knows what she's doing when changing that parameter in the command line. Also note that after this PR most probably that custom fee won't be ever used (as it was mainly to support the transition between fees, let's see!).